examples/separate-config/js/every-feature-multiple-dest-single-ms.js

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
This shows that you can now have multiple destinations without providing a separate move callback, using the new data-context attribute.